changelog/9045.removal.md

```diff
@@ -0,0 +1,10 @@
+Setting `model_confidence=linear_norm` is deprecated and will be removed in Rasa Open Source `3.0.0` .
+
+Rasa Open Source `2.3.0` introduced `linear_norm` as a possible value for `model_confidence`
+parameter in machine learning components such as `DIETClassifier`, `ResponseSelector` and `TEDPolicy`.
+Based on user feedback, we have identified multiple problems with this option and hence
+using `model_confidence=linear_norm` is now deprecated and
+will be removed in Rasa Open Source `3.0.0` . If you were using `model_confidence=linear_norm` for any of the mentioned components,
+it is recommended to revert it back to `model_confidence=softmax` and re-train the assistant. Post re-training,
+it is recommended to [tune the thresholds for fallback components](./fallback-handoff.mdx#fallbacks) as well.
```

Comment:
Let's use the active voice: "it is recommended" -> "we recommend"

"Post re-training," -> "After re-training," 

"as well" <- you can omit this.